mozilla_aws_cli/sts_conn.py

```diff
@@ -97,6 +98,11 @@ def get_credentials(bearer_token, id_token_dict, role_arn):
             {'sts': 'https://sts.amazonaws.com/doc/2011-06-15/'})
         credentials = dict([
             (strip_xmlns(x.tag), x.text) for x in credential_children])
+        if 'Expiration' in credentials:
+            utc_time = datetime.datetime.strptime(
+                credentials['Expiration'], "%Y-%m-%dT%H:%M:%SZ")
+            epoch_time = int((utc_time - datetime.datetime(1970, 1, 1)).total_seconds())
```

Comment:
Use `time.time()`?

Reply:
So `time.time()` gives the epoch time for right now. Are you thinking use `time.time()` in some way in place of `datetime.datetime(1970, 1, 1)` or something else? Tell me more

Reply:
Yes, `time.time()` is how you should probably do `datetime.datetime(1970, 1, 1)`.  :)

Reply:
But `time.time()` gives the epoch time for right now, instead of the epoch time for the beginning of the epoch. I may be missing something.

Reply:
Oh, you're right. You're trying to get the time since the epoch for a given time? I guess I would probably use `datetime.datetime.utcfromtimestamp(0)` then, but they may be the same thing.  :)

Reply:
I like `datetime.datetime.utcfromtimestamp(0)` more as well. I'll change that.

Reply:
Fixed in 2bd30a6